src/server/contentSecurityPolicy.js

```diff
@@ -29,6 +29,7 @@ const connectSrc = (() => {
     'https://cdn.jsdelivr.net',
     'https://widget-mediator.zopim.com',
     'wss://widget-mediator.zopim.com',
+    'https://cors-anywhere.herokuapp.com',
```

Comment:
Kanskje denne passer bedre sammen med de andre lokale tingene i if-statementen under om den bare brukes under utvikling / lokalt?

Reply:
Ja, var usikker på denne. Tenker du på under linje 34 der det er en sjekk på  `if (process.env.NODE_ENV === 'development') {` ?

Reply:
Enig. Dette trengs ikkje i standardoppsettet, kun for lokal utvikling.